spec/message.abnf

```diff
@@ -14,9 +14,9 @@ local-declaration = local s variable [s] "=" [s] expression
 
 quoted-pattern    = "{{" pattern "}}"
 
-matcher           = match-statement 1*([s] variant)
-match-statement   = match 1*([s] selector)
-selector          = expression
+matcher           = match-statement s variant *([s] variant)
```

Comment:
I don't like the inconsistency of whitespace being required before the first match variant and optional before later ones, but it is necessary to support unquoted variant keys as in
```
.input{$count :number}.match $count␣1{{1 new message}}*{{{$count} new messages}}
```